[2023-07-12T18:28:41.265+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: thyroid_preprocessing_DAG.encode_y_train manual__2023-07-12T18:28:05.760737+00:00 [queued]>
[2023-07-12T18:28:41.297+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: thyroid_preprocessing_DAG.encode_y_train manual__2023-07-12T18:28:05.760737+00:00 [queued]>
[2023-07-12T18:28:41.297+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-12T18:28:41.325+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): encode_y_train> on 2023-07-12 18:28:05.760737+00:00
[2023-07-12T18:28:41.340+0000] {standard_task_runner.py:57} INFO - Started process 892 to run task
[2023-07-12T18:28:41.358+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'thyroid_preprocessing_DAG', 'encode_y_train', 'manual__2023-07-12T18:28:05.760737+00:00', '--job-id', '909', '--raw', '--subdir', 'DAGS_FOLDER/dagfile.py', '--cfg-path', '/tmp/tmpzbk8j_xa']
[2023-07-12T18:28:41.359+0000] {standard_task_runner.py:85} INFO - Job 909: Subtask encode_y_train
[2023-07-12T18:28:41.514+0000] {task_command.py:410} INFO - Running <TaskInstance: thyroid_preprocessing_DAG.encode_y_train manual__2023-07-12T18:28:05.760737+00:00 [running]> on host 93aa310e5d41
[2023-07-12T18:28:41.884+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='damirbogdan39@gmail.com' AIRFLOW_CTX_DAG_OWNER='aiflow' AIRFLOW_CTX_DAG_ID='thyroid_preprocessing_DAG' AIRFLOW_CTX_TASK_ID='encode_y_train' AIRFLOW_CTX_EXECUTION_DATE='2023-07-12T18:28:05.760737+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-12T18:28:05.760737+00:00'
[2023-07-12T18:28:41.947+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/preprocessing/_label.py:115: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)

[2023-07-12T18:28:41.950+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/preprocess/y_encoders.py", line 20, in encode_y_train
    y_train_encoded.to_csv(y_train_path, index=False)
AttributeError: 'numpy.ndarray' object has no attribute 'to_csv'
[2023-07-12T18:28:41.969+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=thyroid_preprocessing_DAG, task_id=encode_y_train, execution_date=20230712T182805, start_date=20230712T182841, end_date=20230712T182841
[2023-07-12T18:28:42.038+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 909 for task encode_y_train ('numpy.ndarray' object has no attribute 'to_csv'; 892)
[2023-07-12T18:28:42.112+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-12T18:28:42.168+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
